src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -644,7 +644,7 @@ private void printOptionParamUsage(final StringBuilder sb, final String name, fi
         }
 
         int numSpaces = OPTION_COLUMN_WIDTH - optionLabel.length();
-        if (optionLabel.length() > OPTION_COLUMN_WIDTH) {
+        if (optionLabel.length() > OPTION_COLUMN_WIDTH - 1) {
```

Comment:
Does this only happen in the legacy parser? I guess that for concordance, it should be modified in the `CommandLineArgumentParser`

Reply:
Can you make this change in the other parser as well ?